deployment/terraform/iam.tf

```diff
@@ -80,10 +83,10 @@ resource "aws_iam_role_policy_attachment" "ecs_policy" {
   policy_arn = "${var.aws_ecs_service_role_policy_arn}"
 }
 
-resource "aws_iam_role_policy" "ses_send_email" {
-  name   = "SESSendEmail"
+resource "aws_iam_role_policy" "ecs_ses_send_email" {
```

Comment:
I don't think this one is necessary. This role is assumed by the ECS service to make API calls on our behalf. It shouldn't need to send emails.